integration_tests/symbolics_01.py

```diff
@@ -0,0 +1,11 @@
+from sympy import Symbol, pi
+from lpython import S
+
+def main0():
+    x: S = Symbol('x')
+    y: S = Symbol('y')
+    x = pi
+    z: S = x + y
+    print(z)
```

Comment:
I think, we need to use assert to test the output, something like `assert str(z)  == 'y + pi'`.
This can be done in the subsequent PRs.